plugins/rebalance/init.go

```diff
@@ -0,0 +1,62 @@
+package rebalance
+
+import (
+	"github.com/gluster/glusterd2/glusterd2/servers/rest/route"
+	"github.com/prashanthpai/sunrpc"
+)
+
+// Plugin is a structure which implements GlusterdPlugin interface
+type Plugin struct {
+}
+
+// Name returns name of plugin
+func (p *Plugin) Name() string {
+	return "rebalance"
+}
+
+// SunRPCProgram returns sunrpc program to register with Glusterd
+func (p *Plugin) SunRPCProgram() sunrpc.Program {
+	return nil
+}
+
+// RestRoutes returns list of REST API routes to register with Glusterd
+func (p *Plugin) RestRoutes() route.Routes {
+	return route.Routes{
+		route.Route{
+			Name:        "RebalanceStart",
+			Method:      "POST",
+			Pattern:     "/volumes/rebalance/{volname}/start",
+			Version:     1,
+			HandlerFunc: rebalanceStart},
+		route.Route{
+			Name:        "RebalanceStartForce",
+			Method:      "POST",
+			Pattern:     "/volumes/rebalance/{volname}/start/{force}",
+			Version:     1,
+			HandlerFunc: rebalanceStartForce},
+		route.Route{
+			Name:        "RebalanceStop",
+			Method:      "POST",
+			Pattern:     "/volumes/rebalance/{volname}/stop",
+			Version:     1,
+			HandlerFunc: rebalanceStop},
+		route.Route{
+			Name:        "RebalanceStatus",
+			Method:      "GET",
+			Pattern:     "/volumes/rebalance/{volname}/status",
+			Version:     1,
+			HandlerFunc: rebalanceStatus},
+		route.Route{
+			Name:        "RebalanceFix-layout",
+			Method:      "POST",
+			Pattern:     "/volumes/rebalance/{volname}/fix-layout/start",
```

Comment:
Similar to the "force" option, "fix-layout" should be an option being passed to "rebalance/start". And you just have a single handler for start.